src/main/java/teammates/common/datatransfer/attributes/EntityAttributes.java

```diff
@@ -57,4 +58,18 @@ public boolean isValid() {
      * e.g. trim strings
      */
     public abstract void sanitizeForSaving();
+
+    /**
+     * Check the list for empty nodes ("") and remove them if found.
+     * Return the list without empty nodes or empty list if all nodes are empty
+     */
+    public List<String> removeEmptyElements(List<String> errors) {
+        Iterator<String> iterator = errors.iterator();
+        while (iterator.hasNext()) {
+            if (iterator.next().equals("")) {
+                iterator.remove();
+            }
+        }
+        return errors;
+    }
```

Comment:
Recommend making the method pure, i.e, construct a new list instead of modifying the one being passed in. This also allows using an enhanced for loop instead, which is easier to read.

```java
List<String> nonemptyErrors = ...;
for (String error : errors) {
    if (error.isEmpty()) {
        // add to new list
    }
}
```